setup.cfg

```diff
@@ -1,11 +0,0 @@
-# note that only setuptools specific configuration is
-# in this file. Std configuration as defined by pep621
-# and configuration for other tools goes into pyproject.toml
-
-[options.package_data]
-broadbean =
-    py.typed
```

Comment:
and this file is automatically included in the package data?

Reply:
It is now